ccpp/driver/CCPP_driver.F90

```diff
@@ -149,7 +149,7 @@ subroutine CCPP_step (step, nblks, ierr)
       endif
 
       !--- determine if physics diagnostics buckets need to be cleared
-      if (mod(GFS_control%kdt,GFS_control%nszero) == 1) then
+      if ((mod(GFS_control%kdt-1,GFS_control%nszero)+1) == 1) then
```

Comment:
How is that different from
```
if (mod(GFS_control%kdt,GFS_control%nszero) == 0) then
```
?

Reply:
How about:

`if (mod(GFS_control%kdt-1,GFS_control%nszero) == 0) then`

Reply:
@DusanJovic-NOAA that makes sense.  I will test to confirm it gives the answer I expect.

Reply:
@DusanJovic-NOAA's recommendation works.  Thanks!